internal/mode/static/metrics/collectors/nginx.go

```diff
@@ -37,3 +72,15 @@ func getSocketClient(sockPath string) http.Client {
 		},
 	}
 }
+
+func createPlusClient() (*client.NginxClient, error) {
+	var plusClient *client.NginxClient
+	var err error
+
+	httpClient := getSocketClient(nginxPlusAPISock)
+	plusClient, err = client.NewNginxClient(&httpClient, nginxPlusAPIURI)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to create NginxClient for Plus: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to create NginxClient for Plus: %w", err)
```

Reply:
I wonder if there is some sort of a hook I can create to hit me across the head when I do this 🤦🏼‍♀️  🤣